### PR TITLE
Add int16x8 support for RSqrt operator

### PR DESCRIPTION
**Pull request by SaoirseARM**
Hello,

This PR adds int16x8 support for the RSQRT operator.

Thanks,
Saoirse

### CODE REVIEW

**Comment by gbaned** [comment]
Hi @haozha111 Can you please review this PR ? Thank you!

**Review by haozha111 — APPROVED** [review]

**Comment by haozha111** [comment]
Adding @abattery on quantization related changes.

**Comment by gbaned** [comment]
Hi @haozha111 Can you please review this PR ? Thank you!

**Review by haozha111 — APPROVED** [review]

**Comment by gbaned** [comment]
Hi @SaoirseARM Can you please resolve conflicts? Thank you!

**Comment by gbaned** [comment]
Hi @SaoirseARM Can you please resolve conflicts? Thank you!

File: tensorflow/lite/testing/op_tests/elementwise.py [hunk 1]
```diff
@@ -38,13 +36,6 @@ def f(options):
             "quant_16x8": [False],
             "input_range": [[min_value, max_value]],
         },
-        {
-            "input_dtype": [tf.float32],
-            "input_shape": [[], [1], [1, 2], [5, 6, 7, 8], [3, 4, 5, 6]],
-            "fully_quantize": [True],
-            "quant_16x8": [False],
```
Comment (grantjensen) [inline tensorflow/lite/testing/op_tests/elementwise.py:45]: Why is this case being deleted?

**Comment by gbaned** [comment]
Hi @SaoirseARM Can you please check @grantjensen's comments and keep us posted ? Thank you!

**Review by grantjensen — CHANGES_REQUESTED** [review]
Looking at this closely, we actually should clean up this file now that rsqrt can support 16x8. If you look at [this](https://github.com/tensorflow/tensorflow/pull/60424), you'll see that we had to refactor elementwise.py bc we had an op (rsqrt) that supported allow_fully_quantize, but not 16x8. Now that it supports both, we can once again combine them. Linking below what this new file should be.
[elementwise_py.txt](https://github.com/tensorflow/tensorflow/files/12541515/elementwise_py.txt)

**Comment by grantjensen** [comment]
Almost. Like I said, we can remove allow_quant_16x8. PTAL at the txt file I linked above and just use what I have there. It should work with minimal effort. If there is a reason why you are keeping allow_quant_16x8 please justify it.

**Comment by SaoirseARM** [comment]
> Almost. Like I said, we can remove allow_quant_16x8. PTAL at the txt file I linked above and just use what I have there. It should work with minimal effort. If there is a reason why you are keeping allow_quant_16x8 please justify it.

Apologies, I misunderstood your comment. 
This should be fixed now.
Thanks 
- Saoirse

**Review by grantjensen — APPROVED** [review]

**Comment by grantjensen** [comment]
No worries, my original message was a little confusing. We should be able to get your change merged shortly :)

**Pull request merged**